src/Confluent.Kafka/AdminClient.cs

```diff
@@ -123,6 +123,76 @@ private List<DescribeConfigsReport> extractResultConfigs(IntPtr configResourcesP
+
+            return aclsResultsPtrArr.Select(aclResultPtr =>
+                new CreateAclReport 
+                {
+                    Error = new Error(Librdkafka.acl_result_error(aclResultPtr), false)
+                }
+            ).ToList();
+        }
+
+        private List<AclBinding> extractAclBindings(IntPtr aclBindingsPtr, int aclBindingsCnt)
+        {
+            if (aclBindingsCnt == 0) return new List<AclBinding> {};
+            IntPtr[] aclBindingsPtrArr = new IntPtr[aclBindingsCnt];
+            Marshal.Copy(aclBindingsPtr, aclBindingsPtrArr, 0, aclBindingsCnt);
+
+            return aclBindingsPtrArr.Select(aclBindingPtr =>
+                new AclBinding()
+                {
+                    Pattern = new ResourcePattern
+                    {
+                        Type = Librdkafka.AclBinding_restype(aclBindingPtr),
+                        Name = PtrToStringUTF8(Librdkafka.AclBinding_name(aclBindingPtr)),
+                        ResourcePatternType = Librdkafka.AclBinding_resource_pattern_type(aclBindingPtr)
+                    },
+                    Entry = new AccessControlEntry
+                    {
+                        Principal = PtrToStringUTF8(Librdkafka.AclBinding_principal(aclBindingPtr)),
+                        Host = PtrToStringUTF8(Librdkafka.AclBinding_host(aclBindingPtr)),
+                        Operation = Librdkafka.AclBinding_operation(aclBindingPtr),
+                        PermissionType = Librdkafka.AclBinding_permission_type(aclBindingPtr)
+                    }
+                }
+            ).ToList();
+        }
+
+        private DescribeAclsReport extractDescribeAclsReport(IntPtr resultPtr)
+        {
+            var errCode = Librdkafka.event_error(resultPtr);
+            var errString = Librdkafka.event_error_string(resultPtr);
+            var resultAcls = Librdkafka.DescribeAcls_result_acls(resultPtr,
+                                                                out UIntPtr resultAclCntPtr);
+            return new DescribeAclsReport
+            {
+                Error = new Error(errCode, errString, false),
+                AclBindings = extractAclBindings(resultAcls, (int) resultAclCntPtr)
+            };
+        }
+
+        private List<DeleteAclsReport> extractDeleteAclsReports(IntPtr resultPtr)
+        {
+            var resultResponsesPtr = Librdkafka.DeleteAcls_result_responses(resultPtr, out UIntPtr resultResponsesCntPtr);
+
+            IntPtr[] resultResponsesPtrArr = new IntPtr[(int)resultResponsesCntPtr];
+            Marshal.Copy(resultResponsesPtr, resultResponsesPtrArr, 0, (int)resultResponsesCntPtr);
+
+            return resultResponsesPtrArr.Select(resultResponsePtr => {
+                var matchingAcls = Librdkafka.DeleteAcls_result_response_matching_acls(resultResponsePtr,
+                                                                                       out UIntPtr resultResponseAclCntPtr);
```

Comment:
same comment on formatting as above. put resultResponsePtr on a new line, indent it past the = on the prev line, and out UIntPtr resultResponseAclCntPtr immediately after it.